build.gradle

```diff
@@ -171,11 +171,27 @@ subprojects {
     }
 }
 
+/*
```

Comment:
This should probably be inserted at the top if at all? Do we have to include copyright notices for build scripts?